setup.py

```diff
@@ -108,7 +108,12 @@ def cmake_generate(self):
             install_dir += os.path.sep
 
         cmake_args = [
-            '-DPYTHON_EXECUTABLE=' + sys.executable,
+            # Python_EXECUTABLE is important as it tells CMake's FindPython
+            # which Python executable to use. We absolutely want to use the
+            # interpreter that was used to execute the setup.py.
+            # See https://cmake.org/cmake/help/v3.20/module/FindPython.html#artifacts-specification
```

Comment:
The linter is failing the compile checks because this line is too long. Is there an escape to allow long lines to evade the linter?

Reply:
Yep, `# noqa: E501` at the end of the line should do it.

Reply:
Thanks!